deploy/backplane/mtsre/openshift-connectors/10-mtsre-openshift-connectors-admins.SubjectPermissions.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-mtsre-openshift-connectors-admins
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+  - allowFirst: true
+    clusterRoleName: admin
```

Comment:
Should this be `backplane-openshift-connectors-cr-admins` ? I note the ClusterRole is being defined but nothing seems to be using it.

Reply:
Ahh nice catch,
Actually, we'd want both `admin` as well as `backplane-openshift-connectors-cr-admins` to be associated here.
`admin` being the baseline admin permissions suitable for mt-sre team.
`backplane-openshift-connectors-cr-admins` being the extra set of required permissions not covered by the `admin` role.

@Ankit152 can you add 
```
  - allowFirst: true
    clusterRoleName: backplane-openshift-connectors-cr-admins
    namespacesAllowedRegex: (^redhat-openshift-connectors$)
```
as well under the `permissions` section

Reply:
In that case, I am adding `backplane-mtsre-monitoring` as well. 🙂